services/movie_session.py

```diff
@@ -24,6 +26,15 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
     return MovieSession.objects.get(id=movie_session_id)
 
 
+def get_taken_seats(movie_session_id: int) -> List[dict]:
+    return [
+        {"row": row, "seat": seat}
+        for row, seat in MovieSession.objects.filter(
+            id=movie_session_id).values_list(
+            "ticket__row", "ticket__seat")
```

Comment:
it's fine, as for me, but you could also create additional variable to increase readability